flintrock/services.py

```diff
@@ -240,6 +256,16 @@ def health_check(self, master_host: str):
         except Exception as e:
             raise Exception("HDFS health check failed.") from e
 
+    def create_security_groups(self, flintrock_client_cidr: str):
```

Comment:
`create_security_groups()` suggests this method will actually create the groups, which isn't what it does. I think `get_security_groups()` would be a better name.

Reply:
I will rename to `create_security_group_rules` if you don't mind. Personally I find the use of `get...` to be overused in function names, especially coming from Java land.

Reply:
It's just that "create" suggests the rules will actually be deployed to AWS, but this method only produces a list of rules that then get created/deployed later in ec2.py. Hence the suggestion to use "get" or some other verb that doesn't imply actual creation of the resource. But it's not a big deal.